### PR TITLE
Enforce events to start before it ends

### LINKED ISSUE #99: Enforce events to start before they end
Currently events can start after the end time. We should change this to ensure this bug does not happen, as it does not make sense for the event to start after it has ended.

### PR DESCRIPTION
**Pull request by Fluffykan**
closes #99

### CODE REVIEW

File: src/main/java/seedu/address/model/event/exceptions/DuplicateEventException.java
Comment (JovenSoh) [inline src/main/java/seedu/address/model/event/exceptions/DuplicateEventException.java:1]: Just a minor typo

**Review by JovenSoh — APPROVED** [review]
LGTM, important error to catch.

File: src/main/java/seedu/address/model/event/exceptions/DuplicateEventException.java
Comment (Fluffykan) [reply src/main/java/seedu/address/model/event/exceptions/DuplicateEventException.java:1]: fixed

**Review by tzuennn — APPROVED** [review]
LGTM!

**Pull request merged**